jenkins/Jenkinsfile

```diff
@@ -4,7 +4,7 @@ library identifier: 'fh-pipeline-library@master', retriever: modernSCM(
    credentialsId: 'github-api-token-as-username-password'])
 
 pipeline {
-    agent { label 'docker' }
+    agent { label 'jmolet-vm0' }
```

Comment:
Let's revert the changes in this file. We can have the infrastructure changes merged first, and then re-run jenkins on this PR to make sure it's passing before merging it.

Reply:
The infra has been merged and deployed. The new nodes will be called "candlepin-%". This file can be completely reverted.  You may continue to use the 'docker' agent label or the 'candlepin' label.